cmd/functional-test/testcases.txt

```diff
@@ -0,0 +1,49 @@
+www.example.com {{binary}} -silent 
+www.example.com {{binary}} -silent -l test-data/request.txt
+www.example.com {{binary}} -silent -request test-data/raw-request.txt
+www.example.com {{binary}} -silent -response-time
+www.example.com {{binary}} -silent -title
+www.example.com {{binary}} -silent -location
+www.example.com {{binary}} -silent -cname
+www.example.com {{binary}} -silent -cdn
+www.example.com {{binary}} -silent -probe
+www.example.com {{binary}} -silent -no-fallback
+www.example.com {{binary}} -silent -match-code 200, -mc 200
+www.example.com {{binary}} -silent -match-length 100, -ml 100
+www.example.com {{binary}} -silent -match-string xyz, -ms xyz
+www.example.com {{binary}} -silent -match-regex bbb, -mr bbb
+www.example.com {{binary}} -silent -filter-code 401, -fc 401
+www.example.com {{binary}} -silent -filter-length 23, -fl 23
+www.example.com {{binary}} -silent -filter-string
+www.example.com {{binary}} -silent -threads 50
+www.example.com {{binary}} -silent -rate-limit 5
+www.example.com {{binary}} -silent -tls-grab
+www.example.com {{binary}} -silent -pipeline
+www.example.com {{binary}} -silent -http2
+www.example.com {{binary}} -silent -ports
+www.example.com {{binary}} -silent -paths
```

Comment:
missing the paths arguments (either from file or comma separated)

Reply:
Done.